model/util.go

```diff
@@ -164,9 +165,17 @@ func OpenaiGenerateMessages(prompt string, question string, recentMessages []*Ra
 	if err != nil {
 		return nil, err
 	}
+	nonImageHistoryMessage := []*RawMessage{}
+	for _, message := range historyMessages {
+		re := regexp.MustCompile(`<img[^>]*\s+src=["']?([^"'>\s]+)["']?[^>]*>`)
+		match := re.FindStringSubmatch(message.Text)
+		if match == nil {
+			nonImageHistoryMessage = append(nonImageHistoryMessage, message)
+		}
+	}
```

Comment:
DALL-E-3 output image on casibase cannot be accessed through url. Thus, we need to ignore these AI output images to make system running as normal.